test/transaction.test.ts

```diff
@@ -313,7 +313,9 @@ describe("Path Payment", () => {
     const txBuilder = await stellar.transaction({
       sourceAddress: sourceKp,
     });
-    const txn = txBuilder.swap(new NativeAssetId(), usdcAsset, ".1").build();
+    const txn = txBuilder
+      .swap(new NativeAssetId(), new NativeAssetId(), ".1")
```

Comment:
unrelated. Test is failing bc of testnet reset and no offers avail yet. Just using native -> native here instead, we already have path pay tests between 2 different assets, which swap is using under the hood